app/Jobs/ApplicationDeploymentJob.php

```
@@ -1316,6 +1316,44 @@ private function laravel_finetunes()
+        foreach ($installCmds as $cmd) {
+            if (str_contains($cmd, 'composer install') || str_contains($cmd, 'composer update')) {
+                $hasComposerInstall = true;
+                break;
+            }
+        }
+
+        if ($hasComposerInstall) {
+            $variables = data_get($parsed, 'variables', []);
+
+            $envCommands = [];
+            foreach ($variables as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)